### ISSUE TITLE
can't creat label

### ISSUE THREAD

**Issue by Frankyet** [myparcelnl/prestashop#160]
### Module version

1.5.3. till 1.6.0

### PrestaShop version

1.7.8.3 till 1.7.8.6

### PHP version

7.4

### What went wrong?

Unable to create a label even in a freshly installed prestashop. could it be that the store is installed in a subfolder??

![image](https://user-images.githubusercontent.com/22242384/171936510-ecadb86a-fa8b-4101-97ad-4b6194798427.png)

### Reproduction steps

1. Navigate to orderpage
2. Click on creat than export.
3. See error

![image](https://user-images.githubusercontent.com/22242384/171937333-b63ea683-84ee-44ba-b397-ba8bfdf45ec2.png)

**Comment by mariuszsienkiewicz** [comment]
Hi @Frankyet! 

As I can see developers recently fixed the problem with label creation here: #154 but they broke it again by merging this PR: #147 (I think the change from #154 should be reverted).

The only thing you have to do for it to work again is to change line 222 in this file:
https://github.com/myparcelnl/prestashop/blob/b73368a0aed19f594eb94e17050a4a8950aa7d12/src/Factory/Consignment/ConsignmentFactory.php#L219-L224
to this: 
```php
$this->orderData         = OrderLabel::getDataForLabelsCreate($order->getId()); 
```

And if you are curious the problem happens because of this:
https://github.com/myparcelnl/prestashop/blob/1b05d2b0e7a60a0f1248ec77f092f9cadf3ba829/src/Model/OrderLabel.php#L406-L410

**Comment by Frankyet** [comment]
Hello mariuszsienkiewicz,

This is working thanks, i have more issues and will creat a few more bug reports.

**Comment by wthijmen** [comment]
Hi @Frankyet,

This is fixed since the release version [v1.6.1](https://github.com/myparcelnl/prestashop/releases/tag/v1.6.1)
